src/ngraph/frontend/onnx_import/ops_bridge.cpp

```diff
@@ -39,6 +41,20 @@ namespace ngraph
             } // namespace error
 
             NodeVector add(const Node& node) { return op::add(node); }
+            NodeVector constant(const Node& node)
+            {
+                NodeVector output_nodes;
+                for (const auto& attribute : node.attributes())
+                {
+                    if (!attribute.is_tensor())
+                    {
+                        throw onnx_import::error::attribute::invalid_data{attribute};
```

Comment:
What error message will this generate? Can we Add the name od the node to the error message, so we can see which node caused the problem?

Reply:
No need here, it will be reported at operators bridge level and it knows what node caused the problem.

Reply:
Anyway extracting attribute by name removed this check and exception.